data/changelog.txt

```diff
@@ -1,5 +1,22 @@
-# 0.H
+# 0.H (Herbert? Huxley?)
 
+After over a year, we can finally release 0.H, boasting more changes than any previous release.
+
+## Highlights
+
+Oceans! Travel far enough east and you'll be greeted by a sandy shore.
+Furthermore, the map changes as you travel. The farther east you travel, the bigger cities will be. Cities in the west will start to be farther apart. The forest will get more dense the farther north or west you travel.
+Portal storms are no longer engulfing the entire map and will move. A new boss fight related to portal storms, the void spider, can also be found.
```

Comment:
Void spider isn't related to portal storms

Reply:
And how would you word it?

Reply:
If I knew I would have suggested the change.

Reply:
"In the depths of the earth lurks a Nether horror--fight the new void spider boss if you can find its lair" or something similar?